src/views/AvailableTeams.vue

```diff
@@ -1,12 +1,8 @@
 <template>
   <div class="cont">
-    <div>
-      <h1>Your Teams</h1>
-      <p
-          class="basicText"
-          v-if="myTeams.length === 0">
-        You aren't on a team yet, check out some below!
-      </p>
+    <page-title :returnButton="true"
+                :title="'Your Teams'"
+                :subtitle="subtitle" />
```

Comment:
subtitle?

Reply:
this renders a computed property from a method called subtitle()